### ISSUE TITLE
SPM (release-2.4 branch) warning - incomplete umbrella header

### ISSUE THREAD

**Issue by BlixLT** [core-plot/core-plot#460]
I am trying to add CorePlot as Swift Package, but when building my project I am getting such warnings (see below). As a (hopefully temporary) solution to silence this warning for now I added "`-Wno-incomplete-umbrella`" to other c flags. However, I would prefer not to do that. Is there any plans to do something about it?

```
xxx: Umbrella header for module 'CorePlot' does not include header 'CPTGridLineGroup.h'
xxx: Umbrella header for module 'CorePlot' does not include header 'CPTDerivedXYGraph.h'
xxx: Umbrella header for module 'CorePlot' does not include header 'CPTPlotGroup.h'
xxx: Umbrella header for module 'CorePlot' does not include header 'CPTDebugQuickLook.h'
xxx: Umbrella header for module 'CorePlot' does not include header 'CPTAxisLabelGroup.h'
xxx: Umbrella header for module 'CorePlot' does not include header 'NSNumberExtensions.h'
xxx: Umbrella header for module 'CorePlot' does not include header 'NSDecimalNumberExtensions.h'
xxx: Umbrella header for module 'CorePlot' does not include header 'NSCoderExtensions.h'
xxx: Umbrella header for module 'CorePlot' does not include header 'CPTGridLines.h'

```

**Comment by slizeray** [comment]
Hello, could this issue be fixed? Thanks.

**Comment by eskroch** [comment]
Fixed on the `release-2.4` branch.

**Comment by slizeray** [comment]
@eskroch Thanks a lot.
